modules/mobile/app/controllers/mobile/v0/veterens_affairs_eligibility_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class VeterensAffairsEligibilityController < ApplicationController
+      def show
+        response = mobile_facility_service.get_scheduling_configurations(csv_facility_ids)
+        services = service_adapter.parse(response[:data])
+
+        render json: Mobile::V0::VeterensAffairsEligibilitySerializer.new(@current_user.id, services)
+      end
+
+      private
+
+      def service_adapter
+        Mobile::V0::Adapters::Service.new
+      end
+
+      def mobile_facility_service
+        VAOS::V2::MobileFacilityService.new(@current_user)
+      end
+
+      def csv_facility_ids
+        ids = params.require(:facilityIds)
```

Comment:
We should probably also write a spec for "no facility id provided" just to ensure that it returns 422 or whatever it is.

Reply:
Sounds good. It won't get to the point of calling the upstream service due to the controller requiring the param but still worth putting in there.